.clang-format

```diff
@@ -1,4 +1,6 @@
 # Defines the Chromium style for automatic reformatting.
 # http://clang.llvm.org/docs/ClangFormatStyleOptions.html
 BasedOnStyle: Chromium
-Standard: Cpp20
+Standard: c++20
```

Comment:
```suggestion
Standard: c++23
```
We use cpp23

Reply:
Are you sure clang-format supports that (https://clang.llvm.org/docs/ClangFormatStyleOptions.html#standard) / how would I make that work?

Reply:
You are right.